Sources/Tracing/TracerProtocol.swift

```diff
@@ -60,18 +60,18 @@ public protocol Tracer: LegacyTracer {
         line: UInt
     ) -> Self.Span
 
-    /// Retrieve the active span for the given `ServiceContext`.
+    /// Retrieve the active span for the given service context.
     ///
-    /// - Note: This API does not enable look up of already finished spans.
-    /// It was added retroactively with a default implementation returning `nil` and therefore isn't guaranteed to be implemented by all `Tracer`s.
+    /// - Note: This API does not enable look up of completed spans.
+    /// It was added retroactively with a default implementation returning `nil` and therefore isn't guaranteed to be implemented by all Tracers.
     /// - Parameter context: The context containing information that uniquely identifies the span being obtained.
     /// - Returns: The span identified by the given `ServiceContext` in case it's still recording.
     func activeSpan(identifiedBy context: ServiceContext) -> Span?
 }
 
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)  // for TaskLocal ServiceContext
 extension Tracer {
-    /// Start a new ``Span`` with the given `ServiceContext`.
+    /// Start a new span with the given service context.
```

Comment:
Since this has a default param we can correct this doc to be:


```suggestion
    /// Start a new span with the current, or the explicitly passed, service context.
```